docs/pages/blog/premium-plan-release.md

```diff
@@ -0,0 +1,140 @@
+
+# The new licensing model
+
+As we release the Premium plan, we’d also like to announce its new price and MUI X licensing model.
+
+We’re going for simplicity, restructuring our model to be as clear as possible, based on the recurrent questions we have had on our sales and other channels.
+We’d also like to keep it viable for individual and small teams, while still capturing more of the value we generate for larger companies.
+
+Here is a list of everything that has changed:
+
+> ⚠️ Please note: The following changes apply only to **new customers**. Existing customers are not impacted, and their license agreement remains valid.
+
+- **Price points**
+
+  Balancing our goals and pricing structural changes, we’re lowering our base price points.
+
+  - Pro: $249 → $180 / year
+  - Premium: $599 → $588 / year ($444 - early bird)
+
+- **Removing volume and renewal discounts**
+
+  We aim to give a more transparent price based on the number of developer seats while providing more predictable revenue for MUI X in the long term.
+  We believe that’s meaningful for both our long-standing customers, and MUI as a company.
+
+- **Perpetual in production**
+
+  Customers can use any licensed version perpetually in a production environment (deployed/published applications).
+  But an active license will always be required during development.
+  We roll bug fixes, performance enhancements and other improvements into new releases, so we want to encourage projects in active development to use current versions.
+
+- **Price cap for Premium plan**
+
+  The price for Pro is still capped at 10 seats, but we’re removing the concept of price cap for Premium. We kindly ask you to [contact sales](mailto:sales@mui.com) if you wish to license more than 50 developers.
+
+Please, feel free to read the new [license agreement](https://mui.com/store/legal/mui-x-eula/) in detail.
+
+# What can you expect next?
+
+We mentioned the aggregation functions already,
+but there’s a [lot more coming](https://github.com/mui/mui-x/projects/1), and certainly not only for Premium users.
+We’ll continually enhance all of the components and are increasingly
+putting more effort into usability and overall user and developer experience.
+A product designer is soon joining the team, and we aim to keep exploring the most advanced use cases for data-rich applications,
+while working on bringing the best UX and DX in the market.
+
+We’re very excited about the possibilities the new Premium features will enable, and we hope you’re excited too!
+
+# FAQ 🔧
+
+## How to migrate
+
+To migrate to the premium version in your codebase, install the dedicated package `@mui/x-data-grid-premium` as follows:
+
+With npm:
+
+```js
+npm install @mui/x-data-grid-premium
+```
+
+With yarn:
+
+```js
+yarn add @mui/x-data-grid-premium
+```
+
+All the features from Community and Pro plan are available in the Premium package. You can remove your previous data grid package without any concern.
```

Comment:
> You can remove your previous data grid package without any concern.

Not without first updating imports, as mentioned in the Docs review.

Reply:
After your review, I've changed from "uninstall" to "remove", because I believed it includes removing the imports as well.
But WDYT about
```suggestion
All the features from Community and Pro plan are available in the Premium package. You can remove your previous data grid imports and package without any concern.
``` 
?

Reply:
You can't just remove the imports. They have to be updated.